configure.ac

```diff
@@ -1645,7 +1645,7 @@ if test x$need_bundled_univalue = xyes; then
   AC_CONFIG_SUBDIRS([src/univalue])
 fi
 
-ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --with-bignum=no --enable-module-recovery"
+ac_configure_args="${ac_configure_args} --disable-shared --with-pic --enable-benchmark=no --with-bignum=no --enable-module-recovery --enable-module-schnorrsig --enable-experimental"
```

Comment:
Does it make sense to remove the requirement for the `-enable-experimental` flag to build the schnorrsig module?

Reply:
This was [discussed in the previous PR](https://github.com/bitcoin/bitcoin/pull/17977/files#r475990462).

luke-jr
> Should this be non-experimental before merging?

gmaxwell
> I don't think so: It probably shouldn't be marked non-experimental until after it deployed for activation in Bitcoin because it wouldn't be good to encourage third party users of it while it is still easy to make incompatible changes in Bitcoin (e.g. as was just done. :) )

sipa
> Or, maybe more likely: the API in libsecp256k1 may change still significantly in the near future (e.g. support for variable-length messages, batch validation, ...) even after we treat the scheme itself final.